README.md

```diff
@@ -56,6 +56,15 @@ vars:
 <details><summary>Expand to view details</summary>
 <br>
 
+### Change event date range
+Because of the typical volume of event data, you may want to limit this package's models to work with a recent date range. Therefore we have added the functionality to filter records via the `date_range_start` variable. The default is '2020-01-01' but you may configure it in your root `dbt_project.yml` file:
+
+```yml
+vars:
+    amplitude:
+      date_range_start: '2022-01-01' # your start date here
+```
```

Comment:
I actually think this should be it's own step and not an optional config. Mainly because this will actively filter out the customers data. They should be aware of this config when setting up the package. Can we move this to the step following the database and schema variable declaration?

I am thinking a step 4 and then the additional configs will be 5 and so on.

Reply:
Makes sense to me, updated